lib/v2/fisher-spb/news.js

```diff
@@ -0,0 +1,50 @@
+const got     = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv   = require('iconv-lite');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+
+    const rootUrl  = 'https://fisher.spb.ru/news/';
+    const response = await got({
+        method:       'get',
+        url:          rootUrl,
+        responseType: 'buffer'
+    });
+
+    const $ = cheerio.load(iconv.decode(response.data, 'windows-1251'));
+
+    const descBuilder = (element) => {
+        const content = cheerio.load(`<p>${$('.news-message-text', element).html()}</p>`).root();
+        $('.news-message-media a', element).each((_, elem) => {
+            if ($(elem).hasClass('news-message-youtube')) {
+                content.append(
+                    `<iframe type="text/html" width="640" height="360" src="https://youtube.com/embed/${$(elem)
```

Comment:
Please use templates `art` for any html render

Reply:
done